Release/src/streams/fileio_posix.cpp

```diff
@@ -468,7 +468,7 @@ size_t _getn_fsb(Concurrency::streams::details::_file_info* info,
         if (static_cast<int>(read) > 0)
         {
             auto copy = (std::min)(read, byteCount);
-            auto bufoff = fInfo->m_rdpos - fInfo->m_bufoff;
+            auto bufoff = static_cast<size_t>(fInfo->m_rdpos - fInfo->m_bufoff);
```

Comment:
this is concerning to me since it seems like it could cause truncation on a size passed to memcpy, and may need a validation check.